doc/config.rst

```diff
@@ -14,14 +14,25 @@ With the ``--config`` option you can specify the path to the
 
 The example below will add two rules to the default list of PSR12 set rules:
 
+Consider your codebase as follow:
+
+```
+src/
+    maindir/
+    somedir/
+    otherdir/
+        resources.php
+.php-cs-fixer.php
+```
+
 .. code-block:: php
 
     <?php
 
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
         ->in(__DIR__)
+        ->exclude('somedir')
```

Comment:
example with path/file would be helpful as well

Reply:
indeed, lets add it

Reply:
[See](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6992#discussion_r1205425410).

Reply:
I do not see it. I meant the oposite of `->notPath('bar.php')`, ie. to include a single file like `config.php` which is not inside included `__DIR__ . '/src'`.